example_pipeline/configuration.py

```diff
@@ -190,7 +191,7 @@ def _chebychev_bandpass(lowcut, highcut, fs, rs, order=5):
     'Sleep stage 1 (artefact)' ,
     'Sleep stage R'            ,
     'Sleep stage R (artefact)' ,
-    'undefined'                ,
+    'Sleep stage ?'            ,       # ? = Undefined/unscorable code in Polyman
```

Comment:
Polyman terminology for undefined states adopted in 7059e9ff5136c70de6ca93c1796e626bf4c0e5a0.